extensions/sql-database-projects/package.json

```diff
@@ -155,6 +164,11 @@
         "title": "%sqlDatabaseProjects.changeTargetPlatform%",
         "category": "%sqlDatabaseProjects.displayName%"
       },
+      {
+        "command": "sqlDatabaseProjects.generateProjectFromOpenApiSpec",
```

Comment:
can we add this command to the view/title menu, it would improve discoverability considerably.

![2021-09-16_15-46-38 (1)](https://user-images.githubusercontent.com/12227241/133695154-b4e9b7c6-2322-4114-8123-25c355771832.gif)

Reply:
does this command also need to be added in the command palette section?

Reply:
Don't think so; commands automatically show up in the command palette unless you opt them out in that section of package.config